package.json

```diff
@@ -64,7 +64,7 @@
     "flow-bin": "0.121.0",
     "flow-copy-source": "2.0.9",
     "husky": "4.2.3",
-    "jest": "25.1.0",
+    "jest": "23.6.0",
```

Comment:
Jest V24+ forces babel7 and this project still uses babel6. I downgraded to get tests working.